Source/ARTConnection.m

```diff
@@ -284,6 +262,32 @@ - (void)off:(ARTEventListener *)listener {
     [_eventEmitter off:listener];
 }
 
+-(NSString *)recoveryKey{
+    return [self createRecoveryKey];
+}
+
+- (NSString *)createRecoveryKey {
```

Comment:
Why not to put this implementation inside `recoveryKey` method, since it just return `createRecoveryKey` result? Why `createRecoveryKey` is public?

Reply:
`createRecoveryKey` is still spec'd and the new way. `recoveryKey` was brought back for backward compatibility. See https://github.com/ably/specification/pull/119